src/pages/Details/Details.scss

```
@@ -1 +1,343 @@
+
+          .price {
+            @include text-h1;
+            display: inline;
+          }
+        }
+      }
+    }
+  }
+
+  .product-description {
+    @include flex-center;
+    margin-top: 25px;
+
+    .desc-left {
+      width: 40rem;
+
+      .facts {
+        margin: 10px;
+
+        label {
+          display: block;
+          text-align: center;
+          @include text-h2;
+        }
+
+        .table {
+          @include flex-center;
+          @include text-h4;
+
+          .col-1 {
+            margin: 25px 20px;
+            padding-right: 15px;
+            border-right: 1px solid var(--detail-bordergray);
+            text-align: right;
+          }
+
+          li {
+            margin: 20px 10px;
+            font-weight: 300;
+            color: black;
+          }
+        }
+      }
+    }
+
+    .tasting-info {
+      @include align-middle;
+      align-items: center;
+      margin: 15px 15px;
+
+      label {
+        margin: 20px 0;
+        @include text-h2;
+      }
+
+      .tasting-card-all {
+        @include flex-center;
+
+        .flavor-card {
+          @include align-middle;
+          width: 6.7rem;
+          margin: 10px 10px;
+          padding: 5px 15px 10px 15px;
+          border: 1px solid var(--detail-bordergray);
+          border-radius: 10px;
+
+          .flavor {
+            text-align: center;
+            @include text-h4;
+            color: var(--detail-mediumgray);
+          }
+
+          .flavor-points {
+            padding: 5px 0 10px 0;
+            text-align: center;
+            @include text-h3;
+            color: var(--main-yellow);
+          }
+        }
+      }
+    }
+
+    .aside {
+      width: 35rem;
+
+      .add-to-cart {
+        display: inline-block;
+        position: sticky;
+        width: 25rem;
+        height: 40rem;
+        top: 20px;
+        margin: 40px 10px;
+        padding: 35px 40px;
+        border-radius: 7px;
+        border: 1px solid #dbd9d6;
+
+        .opt-header {
+          @include aside-label-style;
+          padding: 5px 5px;
+          text-align: end;
+          @include text-h4;
+        }
+
+        .options {
+          padding: 0 5px;
+
+          .opt-wrapper {
+            @include aside-border-style;
+          }
+        }
+
+        .amount {
+          .select-amount {
+            display: flex;
+            align-items: center;
+            justify-content: space-evenly;
+            @include aside-border-style;
+
+            .amount-count {
+              margin: 8px 0;
+              @include text-h4;
+            }
+
+            button {
+              border: none;
+              background-color: transparent;
+              @include text-h3;
+              color: var(--main-yellow);
+            }
+          }
+        }
+
+        .total-price-wrapper {
+          .total-price {
+            @include aside-border-style;
+            text-align: center;
+            @include text-h4;
+          }
+        }
```

```suggestion
          .total-price {
            @include aside-border-style;
            text-align: center;
            @include text-h4;
          }
```
.total-price-wrapper 자체에는 아무런 스타일이 없어서 불필요한 nesting이 되는 것 같습니다